definitions/npm/lodash_v4.x.x/flow_v0.63.x/lodash_v4.x.x.js

```diff
@@ -746,8 +746,8 @@ declare module "lodash" {
     shuffle<T>(array: ?Array<T>): Array<T>;
     shuffle<V, T: Object>(object: T): Array<V>;
     size(collection: $ReadOnlyArray<any> | Object | string): number;
-    some<T>(array: ?$ReadOnlyArray<T>, predicate?: Predicate<T>): boolean;
```

Comment:
By making the `null` case overload first, the null case test passes successfully.